doc/user_doc/vic_admin/vicadmin_status_ref.md

```diff
@@ -0,0 +1,91 @@
+# Virtual Container Host Status Reference #
+
+The Web-based administration portal for virtual container hosts, VCH Admin, presents status information about a virtual container host.
+
+If the vSphere environment in which you are deploying a virtual container host does not meet the requirements, the deployment does not succeed. However, a successfully deployed virtual container host can stop functioning if the vSphere environment changes after the deployment. If environment changes adversely affect the virtual container host, the status of the affected component changes from green to yellow or red.
+
+## Virtual Container Host (VCH) ##
+
+VCH Admin checks the status of the processes that the virtual container host runs:
+
+- The port layer server, that ports Docker commands to vSphere API calls.
+- VIC Admin server, that 
+- The vSphere Integrated Containers Engine initialization service, that initializes virtual container hosts. 
+- The Docker engine server, that runs the vSphere Integrated Containers Engine implementations of Docker commands.
+
+**Error message**
+
+The Virtual Container Host status is yellow.
+
+**Cause** 
+
+One or more of the virtual container host processes is not running correctly.
+
+**Solution** 
+
+1. In the VCH Admin portal for the virtual container host, click the link for the **VIC Admin Server** log.
+2. Search the log for references to the different virtual container host processes.
+
+  The different processes are identified in the log by the following names:
+
+  - `port-layer-server`
+  - `vicadmin`
+  - `vic-init`
+  - `docker-engine-server`
+
+3. Identify the process or processes that are not running correctly and attempt to remediate the issues as required.
+
+## Network Connectivity ##
+
+VCH Admin checks external network connectivity by attempting to connect from the virtual container host to docker.io and google.com. VCH Admin only checks the external network connection. It does not check other networks, for example the bridge, management, client, or container networks.
+
+**Error message**
+
+The Network Connectivity status is red.
+
+**Cause** 
+
+The external network connection is down.
+
+**Solution** 
+
+1. In the VCH Admin portal for the virtual container host, click the link for the **VIC Admin Server** log.
+2. Search the log for references to network issues.
+3. In the vSphere Web Client, remediate the network issues as required.
+
+## Firewall ##
+
+VCH Admin checks that the firewall is correctly configured on the ESXi host or the ESXi hosts in the cluster on which the virtual container host is running.
+
+**Error message**
+
+The Firewall status is yellow and shows the error `Firewall must permit 2377/tcp outbound to use VIC`.
+
+**Cause**
+
+The firewall on the ESXi host on which the virtual container host is running no longer allows outbound connections on port 2377.
```

Comment:
This and the license check can also fail if the `management` network is down, or if the endpointVM is unable to connect to vSphere.